### PR TITLE
Update the_model.rst

### PR DESCRIPTION
**Pull request by ralfsideburn**
I had to add these lines. Otherwise I got the "Fixture requires a PHPCR ODM DocumentManager instance, instance of '$class' given." Exception when loading the fixtures with doctrine:phpcr:fixtures:load.

### CODE REVIEW

File: quick_tour/the_model.rst [hunk 1]
```diff
@@ -93,6 +93,9 @@ by creating a new class in the AcmeMainBundle::
     use Doctrine\Common\Persistence\ObjectManager;
     use Doctrine\Common\DataFixtures\FixtureInterface;
     use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+
```
Comment (dantleech) [inline quick_tour/the_model.rst]: Could you remove the empty line?

File: quick_tour/the_model.rst [hunk 2]
```diff
@@ -93,6 +93,9 @@ by creating a new class in the AcmeMainBundle::
     use Doctrine\Common\Persistence\ObjectManager;
     use Doctrine\Common\DataFixtures\FixtureInterface;
     use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+    
+    use Doctrine\ODM\PHPCR\DocumentManager;
+    use Symfony\Cmf\Bundle\SimpleCmsBundle\Doctrine\Phpcr\Page;
```
Comment (WouterJ) [inline quick_tour/the_model.rst]: this one already exists in the example that introduced it (just below this one). The DocumentManager should be added in that example and these 2 should be removed.

Comment (ralfsideburn) [inline quick_tour/the_model.rst]: Where should that be? I can't find it. The only example before this one is "The Big Picture".

Comment (WouterJ) [inline quick_tour/the_model.rst]: belong is after this example, on line 117

**Comment by dbu** [comment]
fixed in 1d1cfaa - thanks a lot ralf for bringing this up!

**Pull request closed without merging**